gk/main.c

```diff
@@ -1220,10 +1438,10 @@ process_pkts_front(uint16_t port_front, uint16_t port_back,
 
 /* Process the packets on the back interface. */
 static void
-process_pkts_back(uint16_t port_back, uint16_t port_front,
+process_pkts_back(uint8_t port_back, uint8_t port_front,
```

Comment:
Why are we updating these types? If we should update them, why aren't we updating the same parameters in `process_pkts_front()`?

Reply:
They shouldn't be updated here, it was my fault that I chose the wrong one when dealing with conflicts.